package.json

```diff
@@ -69,8 +69,7 @@
     "projects": [
       "<rootDir>/packages/core",
       "<rootDir>/packages/signal-client",
-      "<rootDir>/packages/signal-server",
-      "<rootDir>/packages/examples/grid"
+      "<rootDir>/packages/signal-server"
```

Comment:
We're no longer running grid tests... intentional?

Reply:
Not any more, I've restored this.

Reply:
ac4d5d6414a89b18863aec298a8db01ea0c1a9fe